source/chapters/appendix.rst

```diff
@@ -41,7 +35,7 @@ There are a variety of other resources where you can find out more:
 
 * **Mixxx Translations**: To translate Mixxx and promote your mother tongue, go
   to: `<https://www.transifex.com/mixxx-dj-software/public/>`_. Please
-  read the `Translation FAQ <https://github.com/mixxxdj/mixxx/wiki/internationalization>`_
+  read the `Translation FAQ <https://mixxx.org/wiki/doku.php/internationalization>`_
```

Comment:
please revert that, it's unrelated.
Even though we have rediection in pace it's better to have the correct links here.